README.md

```diff
@@ -1,12 +1,65 @@
 # ngrok-go
 
-**Note: This is alpha-quality software. Interfaces are subject to change
-without warning**
-
 This is the ngrok agent in library form, suitable for integrating directly into
 an application. See `example/http.go` for example usage, or the tests in
 `online_test.go`.
 
+## Installation
+
+The best way to install the ngrok agent SDK is through `go get`.
+
+```sh
+go get github.com/ngrok/ngrok-go
+```
+## Documentation
+A quickstart guide and a full API reference are included in the [ngrok go sdk documentation on pkg.go.dev](https://pkg.go.dev/github.com/ngrok/ngrok-go). Check out the [ngrok Documentation](https://ngrok.com/docs) for more information about what you can do with ngrok.
+## Quickstart
+For more examples of using ngrok-go, check out the [/examples](/examples) folder.
+
+The following example uses ngrok to start an http endpoint with a random url that will route traffic to the handler. The ngrok URL provided when running this example is accessible by anyone with an internet connection. 
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/ngrok/ngrok-go"
```

Comment:
`golang.ngrok.com/ngrok`, the example should have already been updated if you want to nab the updated version

Reply:
updated!